src/client/plugins/install-prompt.js

```diff
@@ -1,9 +1,13 @@
 window.addEventListener('beforeinstallprompt', (event) => {
   // Read this to get a better understanding of install prompts:
   // https://developers.google.com/web/fundamentals/app-install-banners/
-  ga('send', 'event', 'install', 'available')
+  typeof window.ga !== 'undefined'
```

Comment:
These lines are doing too much. Each line is always only allowed to do 1 thing. Here it's calling a method or assigning a value to a property at the same time. I'd prefer this:
```js
import { googleAnalyticsTrackingId } from '...'

function trackInstallEvent (eventName) {
  if (googleAnalyticsTrackingId) {
    ga('send', 'event', 'install', eventName)
  }
}

window.addEventListener('beforeinstallprompt', (event) => {
  trackInstallEvent('available')
  event.preventDefault()
})

window.addEventListener('appinstalled', () => trackInstallEvent('installed'))
```